drivers/overlay/overlay.go

```diff
@@ -90,6 +94,14 @@ func Init(dc driverapi.DriverCallback, config map[string]interface{}) error {
 		}
 	}
 
+	if port, ok := config[VxlanPortLabel]; ok {
```

Comment:
How are you planning for the user to pass this option ?
User cannot directly pass driver options into the driver config.
How are you testing this change ?